### ISSUE TITLE
Error while creating a refund

### ISSUE THREAD

**Issue by mnaeem76** [XiaoFaye/WooCommerce.NET#476]
I'm unable to create a refund using C# code below:

```
var refund = new OrderRefund();
refund.amount = 50;
refund.reason = "User wants refund.";
var result = wc.Order.Refunds.Add(refund, 1007).Result;

```
The code throws an exception with the message
`{"{\"code\":\"woocommerce_rest_cannot_create_order_refund\",\"message\":\"Refund Error: You need to specify a refund amount.\",\"data\":500}"}`

I don't know why this error is occuring. I am providing the amount of 50 as you can see in the code. It seems to me that there is problem with the WooCommerece.NET library because when I modify the above code to do a non-api refund it creates a refund with amount of 0.00 only.

```
var refund = new OrderRefund();
refund.amount = 50;
refund.reason = "User wants refund.";
refund.api_refund = false;
var result = wc.Order.Refunds.Add(refund, 1007).Result;

```
Running the code above does not throw error but creates a refund of only 0.00 value. 

Can anyone please help me in this situation? The rest of my solution is based on WooCommerce.NET library and I would hate to write code with raw HTTP requests to create refunds.

- Wordpress version 5.4.1, WooCommerce version 4.0.1 and WooCommerce.NET version 0.8.2

- Error message details
code: woocommerce_rest_cannot_create_order_refund
message: Refund Error: You need to specify a refund amount.
data:500

**Comment by mnaeem76** [comment]
Hello again. I just thought I would also post the data I collected from Fiddler
The headers show that amount is not added as part of the payload to WooCommerece REST API.

{"reason":"Just Testing ","refunded_by":2}
![image](https://user-images.githubusercontent.com/65190008/82327716-406c4d00-99f8-11ea-8d1a-02331cbadd7e.png)

Please help resolve this issue. If this is a bug in the WooCommerece.NET api, please resolve this.

**Comment by XiaoFaye** [comment]
@mnaeem76 There is a bug in the code, I have complied a new dll with fix for you to test.
[WooCommerce.NET.0.8.3.Testing.zip](https://github.com/XiaoFaye/WooCommerce.NET/files/4654405/WooCommerce.NET.0.8.3.Testing.zip)

**Comment by mnaeem76** [comment]
> @mnaeem76 There is a bug in the code, I have complied a new dll with fix for you to test.
> [WooCommerce.NET.0.8.3.Testing.zip](https://github.com/XiaoFaye/WooCommerce.NET/files/4654405/WooCommerce.NET.0.8.3.Testing.zip)

Thanks @XiaoFaye the updated assembly works!!! Thanks for resolving this.

**Comment by XiaoFaye** [comment]
Good to hear that! I will include this fix in the next version.

**Comment by XiaoFaye** [comment]
Hi, this is fixed in version 0.8.3.
